src/alchemtest/amber/access.py

```diff
@@ -106,3 +106,19 @@ def load_invalidfiles():
 
     return Bunch(data=data,
                  DESCR=fdescr)
+
+
+def load_single_dHdl_file():
+    """Load a single dHdl file."""
+
+    module_path = dirname(__file__)
+
+    return glob(join(module_path, 'simplesolvated/charge/0.00/ti-*.out'))
+
+
+def load_single_u_nk_file():
+    """Load a single dHdl file."""
+
+    module_path = dirname(__file__)
+
+    return glob(join(module_path, 'bace_CAT-13d~CAT-17a/solvated/vdw/0.0/ti-*.out.bz2'))
```

Comment:
You can achieve loading a single file with
```python
import alchemtest.amber
bace = alchemtest.amber.load_bace_example()

single_u_nk = bace.data['solvated']['vdw'][0]
```

Reply:
Thank you, I completely missed that, I fixed it!